oat-sa/extension-tao-itemqti
# [TAO-10500] R&O Processing - Create new xml renderer

Related to : https://oat-sa.atlassian.net/browse/TAO-10500
 
Replace response processing renderer to render per interaction response processing xml when per interaction response processing enabled
  
#### How to test
- prepare an instance of TAO
- create a test item
- add some interactions which require an answer
- save the item. check response processing XML in save request body
- import item with normal response processing. edit and save it. check response processing xml

More details about response processing XML format: https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/483557803/Authoring+interaction+level+item+scores

#### Dependencies
 
Requires :
 - [ ] https://github.com/oat-sa/extension-tao-itemqti/pull/1539
